libs/verify-android-environment.sh

```diff
@@ -22,6 +22,18 @@ if [ -z "${ANDROID_NDK_ROOT}" ]; then
   exit 1
 fi
 
+if [ -z "${ANDROID_NDK_HOME}" ]; then
+  echo "Environment variable \$ANDROID_NDK_HOME is not set:"
+  echo "Please export ANDROID_NDK_HOME=\$ANDROID_NDK_ROOT for compatibility with the android gradle plugin."
+  exit 1
+else
```

Comment:
Nit: elif

Reply:
Thanks, I always forgot that bash has this available.